web/src/app/pages/terms/terms.component.ts

```diff
@@ -44,7 +44,7 @@ export class TermsComponent implements OnInit {
   async ngOnInit() {
     const text = await this.dataStore.getTermsOfService();
 
-    this.termsOfServiceText = await parse(text.replace(/#NEWLINE#/g, '\n'));
+    this.termsOfServiceText = await parse(text);
```

Comment:
Is the `#NEWLINE#` workaround no longer needed?

Reply:
Yes, because I tried to programmatically insert the text and it didn't remove newline characters. So we don't need to do the same replacement in the Android version.